lib/v2/components/scanner/scanner_screen.dart

```diff
@@ -49,6 +53,11 @@ class _ScannerScreenState extends State<ScannerScreen> {
     return BlocProvider(
       create: (BuildContext context) => widget._scannerBloc,
       child: BlocBuilder<ScannerBloc, ScannerState>(builder: (context, ScannerState state) {
+        if(state.pageState is Stop) {
+          controller.dispose();
```

Comment:
`controller `and `qrkey` can't they be private?

Reply:
yes! done